src/main/java/core/basesyntax/MainApp.java

```diff
@@ -12,5 +12,16 @@
  * - Отправить все машини на работу и остановить работу всех машин.
  */
 public class MainApp {
-
+    public static void main(String[] args) {
+        Machine track1 = new Track();
```

Comment:
```suggestion
        Machine track = new Track();
```
bad variables naming

Reply:
done